canonical-web-and-design/snapcraft.io
# Feature/revision attributes

This is a feature branch.
All changes in here have been previously reviewed individually.

This PR is for the sake of final QA.

### QA
- ./run or demo
- go to releases page of couple of snaps (one arch, multiple archs, launchpad builds, devmode, etc)
- make sure everything works as expected
